packages/core/test/tracing/reactnavigation.ttid.test.tsx

```diff
@@ -80,6 +81,8 @@ describe('React Navigation - TTID', () => {
               data: {
                 'sentry.op': 'ui.load.initial_display',
                 'sentry.origin': SPAN_ORIGIN_AUTO_UI_TIME_TO_DISPLAY,
+                [SPAN_THREAD_ID]: SPAN_THREAD_ID_JAVASCRIPT,
+                [SPAN_THREAD_NAME]: SPAN_THREAD_NAME_JAVASCRIPT,
```

Comment:
When considering what TTID and TTFD measure it includes both JS and main threads. But the majority of the work should be in JS so I think this is OK.

I just want to note this down.